src/js/pages/KanbanBoard/components/KanbanColumn.js

```diff
@@ -116,9 +116,12 @@ class KanbanColumn extends React.Component {
       { published: published, rating: newRating },
       updatedTake => {
       //success callback
-        if (published) {
-          this.props.updateChosenTakeForChunk(updatedTake.id);
-        }
+        this.props.patchChapter(
+          chapterId, 
+          { published: this.chapterPublished() },
+          updatedChapter => {
+            // chapter has been updated
```

Comment:
do we want to do something with this callback? do we need this callback in the function definition in `KanbanPageActions` if we're not going to use it all?

Reply:
@natibekele probably we don't need this callback